tests/scripts/config.sh.example

```diff
@@ -1,8 +1,10 @@
+#!/bin/sh
+
 # these variables must be defined
-PULP_CLI="pulp"
-PULP_BASE_URL="http://localhost:8080"
-PULP_USER="admin"
-PULP_PASSWORD="password"
+export PULP_CLI="pulp"
+export PULP_BASE_URL="http://localhost:8080"
+export PULP_USER="admin"
+export PULP_PASSWORD="password"
```

Comment:
Also, here it was intended to _not_ export the variables. We don't want the cli to load the values from the environment (yet).